spring-cloud-starter-stream-sink-cassandra/README.adoc

```diff
@@ -24,25 +24,35 @@ A JSON String or byte array representing the entity (or a list of entities) to b
-$$cassandra.cluster.skip-ssl-validation$$:: $$The flag to validate the Servers' SSL certs$$ *($$Boolean$$, default: `$$false$$`)*
-$$cassandra.cluster.use-ssl$$:: $$The flag to use SSL to connect$$ *($$Boolean$$, default: `$$false$$`)*
-$$cassandra.cluster.username$$:: $$The username for connection.$$ *($$String$$, default: `$$<none>$$`)*
-$$cassandra.consistency-level$$:: $$The consistencyLevel option of WriteOptions.$$ *($$ConsistencyLevel$$, default: `$$<none>$$`, possible values: `ANY`,`ONE`,`TWO`,`THREE`,`ALL`,`LOCAL_ONE`,`SERIAL`,`LOCAL_SERIAL`,`QUORUM`,`LOCAL_QUORUM`,`EACH_QUORUM`)*
-$$cassandra.ingest-query$$:: $$The ingest Cassandra query.$$ *($$String$$, default: `$$<none>$$`)*
-$$cassandra.query-type$$:: $$The queryType for Cassandra Sink.$$ *($$Type$$, default: `$$<none>$$`, possible values: `INSERT`,`UPDATE`,`DELETE`,`STATEMENT`)*
-$$cassandra.retry-policy$$:: $$The retryPolicy option of WriteOptions.$$ *($$CassandraRetryPolicy$$, default: `$$<none>$$`, possible values: `DEFAULT`,`DOWNGRADING_CONSISTENCY`,`FALLTHROUGH`,`LOGGING`)*
-$$cassandra.statement-expression$$:: $$The expression in Cassandra query DSL style.$$ *($$Expression$$, default: `$$<none>$$`)*
-$$cassandra.ttl$$:: $$The time-to-live option of WriteOptions.$$ *($$Integer$$, default: `$$0$$`)*
+$$cassandra.cluster.skip-ssl-validation$$:: $$Flag to validate the Servers' SSL certs$$ *($$Boolean$$, default: `$$false$$`)*
+$$cassandra.consistency-level$$:: $$The consistency level for write operation.$$ *($$ConsistencyLevel$$, default: `$$<none>$$`, possible values: `ANY`,`ONE`,`TWO`,`THREE`,`QUORUM`,`ALL`,`LOCAL_QUORUM`,`EACH_QUORUM`,`SERIAL`,`LOCAL_SERIAL`,`LOCAL_ONE`)*
+$$cassandra.ingest-query$$:: $$Ingest Cassandra query.$$ *($$String$$, default: `$$<none>$$`)*
+$$cassandra.query-type$$:: $$QueryType for Cassandra Sink.$$ *($$Type$$, default: `$$<none>$$`, possible values: `INSERT`,`UPDATE`,`DELETE`,`STATEMENT`)*
+$$cassandra.statement-expression$$:: $$Expression in Cassandra query DSL style.$$ *($$Expression$$, default: `$$<none>$$`)*
+$$cassandra.ttl$$:: $$Time-to-live option of WriteOptions.$$ *($$Integer$$, default: `$$0$$`)*
+$$spring.data.cassandra.cluster-name$$:: $$Name of the Cassandra cluster.$$ *($$String$$, default: `$$<none>$$`)*
+$$spring.data.cassandra.compression$$:: $$Compression supported by the Cassandra binary protocol.$$ *($$Compression$$, default: `$$none$$`, possible values: ``,`snappy`,`lz4`)*
+$$spring.data.cassandra.connect-timeout$$:: $$Socket option: connection time out.$$ *($$Duration$$, default: `$$<none>$$`)*
+$$spring.data.cassandra.consistency-level$$:: $$Queries consistency level.$$ *($$ConsistencyLevel$$, default: `$$<none>$$`, possible values: `ANY`,`ONE`,`TWO`,`THREE`,`QUORUM`,`ALL`,`LOCAL_QUORUM`,`EACH_QUORUM`,`SERIAL`,`LOCAL_SERIAL`,`LOCAL_ONE`)*
```

Comment:
Perhaps from:

> Queries consistency level

to 

> Consistency level for the queries

Reply:
Well, that's not what we control here in this project. 
All the `spring.data.cassandra` properties come from Spring Boot: https://github.com/spring-projects/spring-boot/blob/master/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/cassandra/CassandraProperties.java

Here we just white-list them in the `spring-configuration-metadata-whitelist.properties`

Feel free to suggest such a change over there! 😄